lib/sycamore/pyproject.toml

```diff
@@ -80,6 +80,7 @@ transformers = { version = "^4.43.1", optional = true }
 # Legacy partitioner dependencies
 unstructured = { version = "0.10.20", optional = true }
 python-pptx = {version = "^0.6.22", optional = true }
+typing-extensions = "^4.12.2"
```

Comment:
not sure we care overly much about pyproject.toml organization but this is in legacy partitioner dependencies segment

Reply:
Good call. Will move.